recipes/confu_json/all/conanfile.py

```diff
@@ -34,36 +32,42 @@ def _minimum_compilers_version(self):
 
        
     def configure(self):
-        if is_msvc(self) and Version(self.version) < "0.0.9":
-            raise ConanInvalidConfiguration(
-                "Visual Studio is not supported in versions before confu_json/0.0.9")
         if self.settings.compiler == "apple-clang":
             raise ConanInvalidConfiguration(
                 "apple-clang is not supported. Pull request welcome")
-        if self.settings.compiler.get_safe("cppstd"):
-            tools.build.check_min_cppstd(self, self._minimum_cpp_standard)
-
-        min_version = self._minimum_compilers_version.get(
-            str(self.settings.compiler))
-        if not min_version:
-            self.output.warn("{} recipe lacks information about the {} "
-                             "compiler support.".format(
-                                 self.name, self.settings.compiler))
+        if Version(self.version) >= "1.0.0":
+            if self.settings.compiler.get_safe("cppstd"):
+                check_min_cppstd(self, 17)
+            if is_msvc(self) and Version(self.settings.compiler.version) < 17:
+                raise ConanInvalidConfiguration(
+                    "Visual Studio is not supported in versions before 17")
         else:
-            if Version(self.settings.compiler.version) < min_version:
+            if is_msvc(self) and Version(self.version) < "0.0.9":
                 raise ConanInvalidConfiguration(
-                    "{} requires C++{} support. "
-                    "The current compiler {} {} does not support it.".format(
-                        self.name, self._minimum_cpp_standard,
-                        self.settings.compiler,
-                        self.settings.compiler.version))
+                    "Visual Studio is not supported in versions before confu_json/0.0.9")
+            if self.settings.compiler.get_safe("cppstd"):
+                check_min_cppstd(self, 20)
+            min_version = self._minimum_compilers_version.get(
+                str(self.settings.compiler))
+            if not min_version:
+                self.output.warn("{} recipe lacks information about the {} "
+                                "compiler support.".format(
+                                    self.name, self.settings.compiler))
+            else:
+                if Version(self.settings.compiler.version) < min_version:
+                    raise ConanInvalidConfiguration(
+                        "{} requires C++{} support. "
+                        "The current compiler {} {} does not support it.".format(
+                            self.name, 20,
+                            self.settings.compiler,
+                            self.settings.compiler.version))
 
     def requirements(self):
         self.requires("boost/1.79.0")
```

Comment:
```suggestion
        self.requires("boost/1.81.0")
```
Is it compatible with Boost 1.81 ? If yes, it would be great keeping the latest version to avoid version conflict with other packages.